secretstores/local/keyring/keyring.go

```diff
@@ -0,0 +1,206 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keyring
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/99designs/keyring"
+
+	"github.com/dapr/components-contrib/secretstores"
+	"github.com/dapr/kit/config"
+	"github.com/dapr/kit/logger"
+)
+
+// BackendType is alias of keyring.BackendType.
+type BackendType = keyring.BackendType
+
+// All currently supported secure storage backends.
+const (
+	SecretServiceBackend BackendType = keyring.SecretServiceBackend // secret-service
```

Comment:
I would not alias here, the old name is meaningful enough.

Reply:
Fine, I will change it.